docs/references/customization/appcharts.md

```diff
@@ -18,6 +18,108 @@ and
 
 Application charts are expected to create the following resources:
 
-  - A `deployment` specifying the pod and container running the application image.
-  - A `service` to route web requests to the application.
-  - An [ingress](lb.md) per specified route to front the service.
+  - A `Deployment`, `StatefulSet`, or similar main controller, specifying the `Pod` and
+    its containers running the application image.
+
+  - A `Service` to route web requests to the application.
+
+  - An [Ingress](lb.md) per specified route to front the `Service`.
+
+:::note
+
+Up to version 1 Epinio allowed only `Deployment` resources as the main controller of
+applications.
+
+:::
+
+:::tip
+
+Read the contents of the standard application chart.
```

Comment:
Can we provide a link for this content?

Reply:
Would https://github.com/epinio/helm-charts/releases/download/epinio-application-0.1.20/epinio-application-0.1.20.tgz
be sensible ? That is the official link in the epinio chart.

We would however have to update it when the chart gets a new version.

Alternative would be a link to the sources, i.e. https://github.com/epinio/helm-charts/tree/main/chart/application

Reply:
the link to sources is good. It's just to give the reader a direction on where to look and be more explicit on what we mean here.

Reply:
Ok. Will do

Reply:
Done, commit 16ac5feb7ff9029ed2590df2277065285c978ae4